okhttp/src/main/java/okhttp3/CacheControl.java

```diff
@@ -350,5 +337,11 @@ public Builder noTransform() {
     public CacheControl build() {
       return new CacheControl(this);
     }
+
+    private Builder setValueAndReturnBuilder(int valueToUpdate, int valueToConvert, TimeUnit timeUnit) {
+      long valueInSeconds = timeUnit.toSeconds(valueToConvert);
+      valueToUpdate = valueInSeconds > Integer.MAX_VALUE ? Integer.MAX_VALUE : (int) valueInSeconds;
```

Comment:
Java passes arguments by value, not reference, so this statement doesn't actually do anything.